packages/frontend/src/utils/project/getContractSection.ts

```diff
@@ -227,3 +253,39 @@ function isContractUnverified(
     (address) => verificationStatus.contracts[address.toString()],
   )
 }
+
+function isAddressUnverified(
+  address: EthereumAddress,
+  verificationStatus: {
+    projects: Record<string, boolean | undefined>
+    contracts: Record<string, boolean | undefined>
+  },
+): boolean {
+  return verificationStatus.contracts[address.toString()] === false
+}
+
+function escrowToProjectContract(
+  escrow: ProjectEscrow,
+): ProjectContract | undefined {
+  // project config was not yet migrated to the new format
+  if (!escrow.newVersion || escrow.hidden) {
```

Comment:
you can get rid of this condition, it is checked in the loop in the function above

Reply:
I left in on purpose because it's not obvious that this is checked above. Either it should be asserted (And then we can get rid of undefined in the sig) or it should be the way i tweaked it.